packages/react/src/components/Notice/index.tsx

```diff
@@ -14,27 +14,38 @@ export interface NoticeProps
   type?: keyof typeof iconTypeMap;
   title: ContentNode;
   icon?: IconType;
+  variant?: 'default' | 'condensed';
   children?: ReactNode;
 }
 
 const Notice = forwardRef<HTMLDivElement, NoticeProps>(
   (
-    { type = 'info', title, icon, children, ...otherProps }: NoticeProps,
+    {
+      type = 'info',
+      title,
+      icon,
+      variant = 'default',
+      children,
+      ...otherProps
+    }: NoticeProps,
     ref
   ) => {
     return (
       <div
         className={classNames('Notice', {
-          [`Notice--${type}`]: type
+          [`Notice--${type}`]: type,
+          [`Notice--${variant}`]: variant
         })}
         ref={ref}
         {...otherProps}
       >
-        <div className="Notice__title">
+        <div className="Notice__icon">
           <Icon type={icon || (iconTypeMap[type] as IconType)} />
-          {title}
         </div>
-        {children && <div className="Notice__content">{children}</div>}
+        <div className="Notice__content-wrapper">
```

Comment:
Since moving to a grid here, we should use something different from a wrapper. Each element can be at the top level:

```jsx
<Icon />
<div className="Notice__title">...</div>
<div className="Notice__content">...</div>
```

From there, we can define the two-column grid and conditionally place the content in the correct grid column:

```css
.Notice {
  display: grid;
  grid-template-columns: auto 1fr;
  gap: var(--space-small)
}

.Notice__content:not(.Notice--condensed) {
  grid-column: 2;
}
```

For the condensed variant, everything will just be a single column so we could even unset the grid properties.